agent/app/agent.go

```diff
@@ -108,6 +110,17 @@ func newAgent(
 		return nil, err
 	}
 
+	var metadataManager containermetadata.MetadataManager
+	if cfg.ContainerMetadataEnabled {
+		log.Debug("Container metadata feature enabled")
```

Comment:
This line is unnecessary.  If you're using it for your own debugging purposes, please move it to `Trace` level.